runner/src/test/java/com/redhat/service/bridge/runner/it/PostgresResource.java

```diff
@@ -0,0 +1,44 @@
+package com.redhat.service.bridge.runner.it;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.testcontainers.containers.PostgreSQLContainer;
+
+import com.github.dockerjava.api.model.ExposedPort;
+import com.github.dockerjava.api.model.PortBinding;
+import com.github.dockerjava.api.model.Ports;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+public class PostgresResource implements QuarkusTestResourceLifecycleManager {
+
+    private final PostgreSQLContainer postgres = new PostgreSQLContainer<>("postgres:10.5")
+            .withDatabaseName("hibernate_orm_test")
+            .withUsername("hibernate_orm_test")
+            .withPassword("hibernate_orm_test")
+            .withExposedPorts(5432)
```

Comment:
Is it possible to move to test application properties file? You can use [config programmatic API](https://quarkus.io/guides/config#programmatically-access-the-configuration) if necessary to retrieve them

Reply:
As per chat I did a quick investigation and unfortunately it does not work. I guess it's because of @QuarkusTestResource.